src/components/MonthlyCalendar/readme.md

```diff
@@ -199,3 +199,379 @@ const initialState = {
+    margin: 8px 4px 0 16px;
+`;
+
+const StyledHourContainer = styled.div`   
+    font-size: 14px;
+    margin-left: 8px;
+`;
+
+const StyledContentContainer = styled.div`
+    display: flex;
+    align-items: center;    
+    font-size: 12px;
+    margin: 0 4px 4px 65px;
+`;
+
+const StyledIconContainer = styled.div`
+    margin-right: 5px;
+`;
+
+const StyledAssignedText = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})
+`
+    font-size: 12px;
+    color: ${props => props.text.header};
+`;
+
+function AssignedTasksBasicInformation(props) {
+    const {assignedTasksCount} = props;
+
+    if (!assignedTasksCount) return null;
+    return (
+        <StyledStatisticsContainer>
+            ASSIGNED
+            <Badge
+                className="rainbow-m-around_medium"
+                label={assignedTasksCount}
+                title="AssignedTasks" />
+        </StyledStatisticsContainer>
+    );
+}
+
+function AvailableTasksBasicInformation(props) {
+    const {availableTasksCount} = props;
+
+    if (!availableTasksCount) return null;
+    return (
+        <StyledStatisticsContainer>
+            AVAILABLE
+            <Badge 
+                className="rainbow-m-around_medium"
+                label={availableTasksCount}
+                title="availableTasks" />
+        </StyledStatisticsContainer>
+    );
+}
+
+function TaskStatus({task}) {
+    if (task.isConfirmed) {
+        return (
+            <StyledContentContainer>
+                <StyledIconContainer>
+                    <FontAwesomeIcon icon={faCheckCircle} size="lg" color="#1de9b6"/>
+                </StyledIconContainer>
+                <StyledAssignedText>Confirmed</StyledAssignedText>
+            </StyledContentContainer>
+        );
+    }
+    return (
+        <StyledContentContainer>
+            <StyledAssignedText>Not confirmed</StyledAssignedText>
+        </StyledContentContainer>
+    );
+}
+
+function AvailableTaskInformation({task}) {
+    return (
+            <StyledCardContainer>
+                <Card>
+                    <StyledHeaderContainer>
+                        <Avatar icon={<FontAwesomeIcon icon={faCalendarAlt} />} />
+                        <StyledHourContainer>{task.hour}</StyledHourContainer>
+                    </StyledHeaderContainer>
+                    <StyledContentContainer>
+                        {task.description}
+                    </StyledContentContainer>
+                    <TaskStatus task={task}/>
+                </Card>
+            </StyledCardContainer>
+    );
+}
+
+function AssignedTaskInformation({task}) {
+    return (
+            <StyledCardContainer>
+                <Card>
+                    <StyledHeaderContainer>
+                        <Avatar icon={<FontAwesomeIcon icon={faCalendarAlt} />} />
+                        <StyledHourContainer>{task.hour}</StyledHourContainer>
+                    </StyledHeaderContainer>
+                    <StyledContentContainer>
```

Comment:
this could be a single reusable component

Reply:
done